src/Geometry.cpp

```diff
@@ -1113,8 +1112,8 @@ void Geometry::initializeCmfd() {
  * @brief Returns a pointer to the map that maps FSR keys to FSR IDs
  * @return pointer to _FSR_keys_map map of FSR keys to FSR IDs
  */
-ParallelHashMap<std::string, fsr_data*>* Geometry::getFSRKeysMap() {
-  return &_FSR_keys_map;
+ParallelHashMap<std::string, fsr_data*> & Geometry::getFSRKeysMap() {
```

Comment:
Looks like the google c++ style guide recommends:

    Pointer operators have no space after the * or &.

So I'll remove the space before `&`.